storage/buckets/main.go

```diff
@@ -1,500 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Looks like this file is still being deleted. Are we sure that won't break any docs?

Reply:
Oops, deleted it by mistake